### PR TITLE
S3 bucket list, response with uploaded storageclass.

### PR DESCRIPTION
**Pull request by taozix**

### CODE REVIEW

File: weed/util/http_header.go [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+	// Signature v2 related constants
+	AmzSignatureV2 = "Signature"
+	AmzAccessKeyID = "AWSAccessKeyId"
+
+	// Response request id.
+	AmzRequestID = "x-amz-request-id"
+
+	// Deployment id.
+	MinioDeploymentID = "x-minio-deployment-id"
```
Comment (chrislusf) [inline weed/util/http_header.go]: please remove unused variables.

File: weed/util/http_header.go [hunk 2]
```diff
@@ -0,0 +1,141 @@
+/*
```
Comment (chrislusf) [inline weed/util/http_header.go]: Move this file to s3api folder

Comment (chrislusf) [reply weed/util/http_header.go]: And remove all unused variables.

Comment (taozix) [reply weed/util/http_header.go]: it seems will cause "import cycle" if move http_header to s3api.

**Comment by chrislusf** [comment]
Create a folder there

**Pull request merged**